resources/src/commands/buns.rs

```diff
@@ -12,7 +12,7 @@ pub async fn run(
 ) -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
     increment_buns(enterprise_user_id).await;
 
-    add_reaction::run(channel, timestamp, "buns")
+    add_reaction::add_reaction(channel, timestamp, "buns")
```

Comment:
`add_reaction::add_reaction` is arguably even worse naming, but these names were already updated in a previous commit, not sure why it's missing here. Please pull the latest code.